cves/CVE-2018-11784.yml

```diff
@@ -206,4 +217,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: Overall this was a coding mistake and specifically an err in assumptions. 
```

Comment:
Do you have any opinions on their mitigation?

Reply:
Not really, it wasn't apparent that they tried multiple things to fix it before it worked out. The bug occurred due to lack of carefulness on the path of the developer. They probably weren't thinking about that bug being a possibility.